easybuild/tools/modules.py

```diff
@@ -864,10 +890,14 @@ def update(self):
                 except (IOError, OSError), err:
                     raise EasyBuildError("Failed to update Lmod spider cache %s: %s", cache_fp, err)
 
-    def prepend_module_path(self, path):
+    def prepend_module_path(self, path, set_mod_paths=True):
```

Comment:
yes, for Lmod the `unuse` is not required (since `use` pushes the path to the front), so this is an optimised version that only works for Lmod...